exporter/exporterhelper/batch_sender.go

```diff
@@ -91,6 +94,7 @@ func (bs *batchSender) Start(_ context.Context, _ component.Host) error {
 		}
 	}()
 
+	bs.logger.Debug("Started Batch Sender", zap.String(obsmetrics.ExporterKey, bs.fullName))
```

Comment:
I think the `obsmetrics.ExporterKey` entry is not necessary, the logger will have a "name" entry (see https://github.com/open-telemetry/opentelemetry-collector/blob/main/service/internal/components/loggers.go#L38) with this.

Reply:
removed the exporter key from all the logs.